opengever/maintenance/scripts/fix_broken_intid_references.py

```diff
@@ -69,21 +79,28 @@ def check(self):
         self.write_broken_paths(broken, self.objs_to_fix_path + '.backup')
 
     def fix(self, commit_every=100):
-        logger.info("Loading broken paths from: {}".format(objs_to_fix_path))
+        logger.info("Loading broken paths from: {}".format(self.objs_to_fix_path))
         broken = self.load_broken_paths(self.objs_to_fix_path)
 
         if not broken:
             logger.info("There is nothing to fix.")
             return
 
         fixed = []
+        query = {'path': {'query': broken, 'depth': 0}}
+        if self.directory_path:
+            query['path']['query'] = self.directory_path
+
```

Comment:
why is this necessary? the previous implementation only queried broken objects which is what we want here.